modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -581,6 +584,34 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
         }
     }
 
+    @Override
+    protected void addOneOfForComposedSchema(Entry<String, Schema> stringSchemaEntry, String modelName, ComposedSchema composedSchema,
+        String nOneOf){
+        if (useOneOfInterfaces) {
```

Comment:
useOneOfInterfaces is always true?

Reply:
yes, you are right useOneOfInterfaces is always true, i will change the code, that there is no if cases anymore.

Reply:
do you mean this?

Reply:
@Scorpioncik please review this pull request again.